testsuite/MDAnalysisTests/converters/test_rdkit.py

```diff
@@ -397,9 +378,36 @@ def test_cache(self):
         new_cache = cached_func.cache_info()
         assert cache == new_cache
 
+    def test_sanitize_fail_warning(self):
+        mol = Chem.MolFromSmiles("[H]-N(-[H])(-[H])-[H]", sanitize=False)
+        mol.UpdatePropertyCache(strict=False)
+        mol = Chem.AddHs(mol)
+        u = mda.Universe(mol)
+        with pytest.warns(UserWarning, match="Could not sanitize molecule"):
+            u.atoms.convert_to.rdkit(NoImplicit=False)
+        with pytest.warns(None) as record:
+            u.atoms.convert_to.rdkit()
+        if record:
+            assert all("Could not sanitize molecule" not in str(w.message)
+                       for w in record.list)
+
 
 @requires_rdkit
 class TestRDKitFunctions(object):
+
+    def mol_to_template(self, mol):
+        """Remove bond orders and charges from molecule"""
+        template = Chem.AddHs(mol)
+        for atom in template.GetAtoms():
+            atom.SetIsAromatic(False)
+            atom.SetFormalCharge(0)
+            atom.SetNoImplicit(True)
+        for bond in template.GetBonds():
+            bond.SetIsAromatic(False)
+            bond.SetBondType(Chem.BondType.SINGLE)
+        template.UpdatePropertyCache(strict=False)
+        return template
```

Comment:
Does template have a special meaning here? If not, could you please rename this to something like `remove_bond_orders_and_charges` or something a bit more descriptive for the tests?